src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -76,21 +78,52 @@ class SurrealPersistenceService implements PersistenceService {
 
     void initializeDb(){
         // create wave_build table
-        final ret1 = surrealDb.sqlAsMap(authorization, "define table wave_build SCHEMALESS")
+        final ret1 = surrealDb.sqlAsMap(authorization, "DEFINE TABLE wave_build SCHEMALESS")
         if( ret1.status != "OK")
             throw new IllegalStateException("Unable to define SurrealDB table wave_build - cause: $ret1")
         // create wave_request table
-        final ret2 = surrealDb.sqlAsMap(authorization, "define table wave_request SCHEMALESS")
+        final ret2 = surrealDb.sqlAsMap(authorization, "DEFINE TABLE wave_request SCHEMALESS")
         if( ret2.status != "OK")
             throw new IllegalStateException("Unable to define SurrealDB table wave_request - cause: $ret2")
         // create wave_scan table
-        final ret3 = surrealDb.sqlAsMap(authorization, "define table wave_scan SCHEMALESS")
+        final ret3 = surrealDb.sqlAsMap(authorization, "DEFINE TABLE wave_scan SCHEMALESS")
         if( ret3.status != "OK")
             throw new IllegalStateException("Unable to define SurrealDB table wave_scan - cause: $ret3")
         // create wave_scan table
-        final ret4 = surrealDb.sqlAsMap(authorization, "define table wave_scan_vuln SCHEMALESS")
+        final ret4 = surrealDb.sqlAsMap(authorization, "DEFINE TABLE wave_scan_vuln SCHEMALESS")
         if( ret4.status != "OK")
-            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret3")
+            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret4")
+
+        try {
+            // create wave_matrics_pull  table
+            surrealDb.sqlAsString(authorization, "DEFINE TABLE wave_metrics_pull SCHEMAFULL;" +
+                                                "DEFINE FIELD count ON TABLE wave_metrics_pull TYPE number;" +
+                                                "DEFINE FIELD ip ON TABLE wave_metrics_pull TYPE string; " +
+                                                "DEFINE FIELD userid ON TABLE wave_metrics_pull TYPE number;" +
+                                                "DEFINE FIELD imagename ON TABLE wave_metrics_pull TYPE string; " +
+                                                "DEFINE FIELD date ON TABLE wave_metrics_pull TYPE datetime;" +
+                                                "DEFINE INDEX indexip ON TABLE wave_metrics_pull COLUMNS ip; " +
+                                                "DEFINE INDEX indexuserid ON TABLE wave_metrics_pull COLUMNS userid;" +
+                                                "DEFINE INDEX indeximagename ON TABLE wave_metrics_pull COLUMNS imagename;")
```

Comment:
A multiline string would make it much more readable

Reply:
done